display/file.go

```diff
@@ -0,0 +1,113 @@
+package display
+
+import (
+	"fmt"
+	"io/ioutil"
+	"mime"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/makeworld-the-better-one/amfora/renderer"
+	"github.com/makeworld-the-better-one/amfora/structs"
+	"github.com/spf13/viper"
+)
+
+func pathFromURI(u string) string {
```

Comment:
```suggestion
func pathFromFileURI(u string) string {
```

This func is available to all of `display/` so changing the name here makes it more clear.